src/modules/app/routes/categories.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import { categoriesFetcher, locationLayoutFetcher } from '../../endpoint/fetchers'
+import { createAction } from 'redux-actions'
+
+import type { Dispatch, GetState } from 'redux-first-router/dist/flow-types'
+import { clearStoreOnCityChange, clearStoreOnLanguageChange } from '../../endpoint/actions/remover'
+
+export const CATEGORIES_ROUTE = 'CATEGORIES'
+
+export const goToCategories = (city: string, language: string, categoryPath: ?string) =>
+  createAction(CATEGORIES_ROUTE)({city, language, categoryPath})
+
+/**
+ * CategoriesRoute, matches /augsburg/de*
+ * @type {{path: string, thunk: function(Dispatch, GetState)}}
+ */
+export const categoriesRoute = {
+  path: '/:city/:language/:categoryPath*',
+  thunk: async (dispatch: Dispatch, getState: GetState) => {
+    const state = getState()
+    const {city, language} = state.location.payload
+    const prev = state.location.prev
+
+    if (prev.payload.language && prev.payload.language !== language) {
+      clearStoreOnLanguageChange(dispatch, getState)
+    }
+
+    if (prev.payload.city && prev.payload.city !== city) {
+      clearStoreOnCityChange(dispatch, getState)
+    }
+
+    await locationLayoutFetcher(dispatch, getState)
```

Comment:
I think we shouldn't load there the data which is neccassary for the location layout. Can we maybe have this a bit more local to the definition of <LocationLayout>?

Reply:
done